packages/bpk-component-theme-switcher/src/theming.js

```diff
@@ -0,0 +1,17 @@
+import {
+  colorGreen100,
+  colorGreen500,
+} from 'bpk-tokens/tokens/base.es6';
+
+const bpkCustomTheme = {
```

Comment:
this is where the hardcoded (for now) custom theme lives, we need to decide colours for the different components

Reply:
It might be better to do something like we have in native, with a `theme` object that all the specific attributes are derived from. Just makes things DRYer for if we decide to use a different colour.

https://github.com/Skyscanner/backpack/blob/master/native/storybook/themeAttributes.js#L19

Not the end of the world though as this might go away in favour of something using the branding API.

Reply:
I sure I'm missing something but I'm importing tokens and and use those values rather than hardcoded one like your example, how is that different? could be something obvious that I'm missing 😄

Reply:
So I was thinking it could be like this:

```
const theme = {
  primaryColor: colorGreen500,
  ...etc
};

const bpkCustomTheme = {
  blockquoteBarColor: primaryColor,
  horizontalNavBarSelectedColor: primaryColor,
  horizontalNavLinkActiveColor: primaryColor,
  horizontalNavLinkColor: primaryColor,
  ...etc
};
```

So then you get two benefits:

1 - DRYer - if we decide that the custom theme should use a different token in future, you only need to change the value of `primaryColor`.

2 - It's closer to how it'll work in real life, as the branding API will return values like `brandPrimaryColor`, `brandAccentColor` etc and then those will be used to create theme attributes.

But it's also not a big deal at all, it's just a minor code style thing after all!